### DISCUSSION TITLE
Impact of Netty ThreadLocals with Virtual Threads (Loom)

### DISCUSSION THREAD

**Discussion by marlonpatrick** [micronaut-projects/micronaut-core#10916] [Q&A]
Hey folks,

Could you help me with a question?

From what I've read, Netty can have problems when working with Virtual Threads due to the use of ThreadLocal.

But, it's not clear to me how this affects Micronaut. 

Is there an impact on the Http server side? 

Is there an impact on the Http cliente side?

Tks

**Comment by sdelamo** [comment]
@yawkat can you provide an answer?
